HW2/ptb-lm.py

```diff
@@ -92,7 +92,7 @@
 
 # NOTE ==============================================
 # This is where your models are imported
-from models import RNN, GRU 
+from models import RNN#, GRU
```

Comment:
remove the change here. Tu devrais pas faire des modifications de meme sur master. Ca brise le code des autres

Reply:
J'ai fait ca pour la meme raison que l'autre